eva/interfaces/relational/db.py

```diff
@@ -23,18 +37,43 @@ def __init__(self, reader, writer):
         self._reader = reader
         self._writer = writer
         self._cursor = None
+        self._result: Batch = None
 
     def cursor(self):
         # One unique cursor for one connection
         if self._cursor is None:
             self._cursor = EVACursor(self)
         return self._cursor
 
+    def load(
+        self, file_regex: str, table_name: str, format: str, **kwargs
+    ) -> EVARelation:
+        return self.cursor().load(file_regex, table_name, format, **kwargs)
+
+    def table(self, table_name: str) -> EVARelation:
+        return self.cursor().table(table_name)
+
+    def query(self, sql_query: str) -> EVARelation:
+        return self.cursor().query(sql_query)
+
+    def df(self) -> pandas.DataFrame:
```

Comment:
Will `self.cursor().query(second_query)` will work even if the first query got a empty result?

Reply:
This is lazy, so you need to call `.df()` on the output to run the query.